app/views/components/details.scala.html

```diff
@@ -0,0 +1,28 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(summaryText: String = "", bodyHtml: Option[Html] = None)
```

Comment:
In every usage second argument is assign to local variable - we can use multiple parameter group and make little less boilerplate.

IFAIR if you had declare this partial as
```
@(summaryText: String = "")(bodyHtml: Html = HtmlFormat.empty)
```
you would use it as 
```
@components.details(messages("site.details.summary_text_this")){ 
   @messages("declaration.type.additionalActorsSummaryText", components.details_content_link())
}
```
CC: @PauloPortugal @PatrykRudnicki

Reply:
That is a good point. Should the `bodyHtml` even be optional?

Reply:
Done